packages/core/config/webpack.development.js

```diff
@@ -9,10 +9,12 @@ const webpackConfig = require('./webpack.common')
 const { mergeUserWebpackConfig } = webpackConfig
 const redwoodConfig = getConfig()
 
+/** @type {import('webpack').Configuration} */
 const baseConfig = merge(webpackConfig('development'), {
   devServer: {
     // https://webpack.js.org/configuration/dev-server/
-    hot: true,
+    hot: redwoodConfig.web.liveReload,
+    watch: redwoodConfig.web.liveReload,
```

Comment:
Why not just pass these in as an option via `yarn rw dev --forward="--hot=true --watch-true"?

Reply:
This was my initial inclination, but none of those options seemed to work: webpack-dev-server has 3 levels of "live reloading."